src/activy/utils/clickSettings.py

```
@@ -0,0 +1,8 @@
+def clickSettings(device):
+    # this has to be done with proportions to click in a specific area where the tab selector is
+    # the challange button is an imageView which has a varries ammount of instances
+    width = device.info["displayWidth"]
+    height = device.info["displayHeight"]
```

Ruff (0.8.2)

Local variable `height` is assigned to but never used

Remove assignment to unused variable `height`

(F841)